client/scss/components/_header.scss

```diff
@@ -164,4 +164,11 @@ header {
     @include media-breakpoint-down(xs) {
         padding-left: $mobile-nav-indent;
     }
+
+    &-icon {
+        vertical-align: text-top;
+        margin-right: 0.5em;
+        width: 1.5em;
+        height: 1.5em;
```

Comment:
This snippet seems to be recurring in the font icon to SVG PRs. I do wonder if we could have a sensible along the lines of:

```css
svg.icon {
    width: 1.5em;
    height: 1.5em;
    vertical-align: text-top;
}
```

then override where needed like here.

Reply:
having played with this a bit, may we provide a few modifiers something like:

```scss
.icon {
    &.initial {
        width: 1em;
        height: 1em;
        vertical-align: initial;
    }

    &.default {
        width: 1.5em;
        height: 1.5em;
        vertical-align: text-top;
    }
}
```

?

Reply:
@zerolab Thanks for the feedback. I added a `svg-icon` mixin. With a default size.